### PR TITLE
i18n(id): translate `frontmatter.md` into Indonesian

### PR DESCRIPTION
**Pull request by nazhard**
#### Description

- What does this PR change? Give us a brief description.

Translate `frontmatter` reference docs into Indonesian.

### CODE REVIEW

File: docs/src/content/docs/id/reference/frontmatter.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+---
+
+Selamat datang di halaman Tentang!
+```
+
+## Bidang Frontmatter
+
+### `title` (diperlukan)
+
+**type:** `string`
```
Comment (HiDeoo) [inline docs/src/content/docs/id/reference/frontmatter.md:21]: Just to double check, this is fine to have "type" here, it translates well and not something that may have been missed?

File: docs/src/content/docs/id/reference/frontmatter.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+---
+title: Halaman tanpa daftar isi
+tableOfContents: false
+---
+```
+
+### `template`
+
+**type:** `'doc' | 'splash'`  
+**default:** `'doc'`
```
Comment (HiDeoo) [inline docs/src/content/docs/id/reference/frontmatter.md:79]: Same question like "type" above to be sure, is this fine to have "default" here? I know for example in French we change it but it may be totally fine in Indonesian.

File: docs/src/content/docs/id/reference/frontmatter.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+# Mengganti teks tautan halaman sebelumnya
+prev: lanjutkan tutorial
+---
+```
+
+```md
+---
+# Mengabaikan tautan dan teks halaman sebelumnya
+prev:
+  link: /unrelated-page/
```
Comment (HiDeoo) [inline docs/src/content/docs/id/reference/frontmatter.md:217]: Not sure if needed, but the link can use translated keywords as Indonesian words may be easier to understand this is a totally unrelated page. Any thoughts?

File: docs/src/content/docs/id/reference/frontmatter.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+  hidden?: boolean;
+  badge?: string | BadgeConfig;
+  attrs?: Record<string, string | number | boolean | undefined>;
+}
+```
+
+#### `label`
+
+**type:** `string`  
+**default:** [`title`](#title-required) halaman
```
Comment (HiDeoo) [inline docs/src/content/docs/id/reference/frontmatter.md:270]:
```suggestion
**default:** [`title`](#title-diperlukan) halaman
```

Slight tweak to fix the anchor link.

File: docs/src/content/docs/id/reference/frontmatter.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+
+Atur label untuk halaman ini di _sidebar_ saat ditampilkan dalam grup tautan yang dibuat secara otomatis.
+
+```md
+---
+title: Tentang proyek ini
+sidebar:
+  label: About
```
Comment (HiDeoo) [inline docs/src/content/docs/id/reference/frontmatter.md:278]: Any thoughts about translating the label? Would it be better in Indonesian?

File: docs/src/content/docs/id/reference/frontmatter.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+Menambahkan _badge_ ke halaman di _sidebar_ saat ditampilkan dalam grup tautan yang dibuat secara otomatis.
+Saat menggunakan string, _badge_ akan ditampilkan dengan warna aksen default.
+Secara opsional, berikan [objek `BadgeConfig`](/id/reference/configuration/#badgeconfig) dengan bidang `text` dan `variant` untuk menyesuaikan _badge_.
+
+```md
+---
+title: Halaman dengan badge
+sidebar:
+  # Menggunakan varian default yang sesuai dengan warna aksen situs Anda
+  badge: New
```
Comment (HiDeoo) [inline docs/src/content/docs/id/reference/frontmatter.md:325]: Same as above, should "New" be translated too?

**Review by HiDeoo — COMMENTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and this is a great work, I only left a few minor questions. Thanks again for your impressive contribution!

File: docs/src/content/docs/id/reference/frontmatter.md (hunk 2 above)
Comment (nazhard) [reply docs/src/content/docs/id/reference/frontmatter.md:79]: Actually, this can be written as "type" and "default". But I will try to translate it for better quality 👍

File: docs/src/content/docs/id/reference/frontmatter.md (hunk 3 above)
Comment (nazhard) [reply docs/src/content/docs/id/reference/frontmatter.md:217]: You're right, maybe I should translate it
